fetch_submissions.py

```diff
@@ -141,13 +145,18 @@ def save_submission_as_file(submission):
 
 
 if __name__ == '__main__':
-    if (len(sys.argv) != 2):
-        print('Usage: python3 fetch_submissions.py <username>')
+    is_logged_in = False
+    for i in range(5):
+      username = input('Username: ')
+      password = getpass.getpass()
+      if successfully_login(username, password):
+        is_logged_in = True
+        break
+      print("The username and/or password you specified are not correct.", file=sys.stderr)
+
+    if not is_logged_in:
+      print(f"You have exceeded the maximum number of login attempts allowed.")
```

Comment:
I'm curious as to why you didn't pipe this to stderr as well

Reply:
Left it out by accident